Apps/W1/Shopify/app/src/Base/Codeunits/ShpfyInstaller.Codeunit.al

```diff
@@ -38,12 +39,14 @@ codeunit 30273 "Shpfy Installer"
 
         RetenPolAllowedTables.AddAllowedTable(Database::"Shpfy Log Entry", LogEntry.FieldNo(SystemCreatedAt));
         RetenPolAllowedTables.AddAllowedTable(Database::"Shpfy Data Capture", DataCapture.FieldNo(SystemModifiedAt));
+        RetenPolAllowedTables.AddAllowedTable(Database::"Shpfy Skipped Record", SkippedRecord.FieldNo(SystemCreatedAt));
```

Comment:
This will not get executed unless a user invokes "Refresh Allowed Tables" because IsInitialSetup will always be false. GetShopifyLogEntryAddedToAllowedListUpgradeTag is already inserted into all upgraded databases.

We should add a new upgrade tag and move skipped records into a new procedure. Since you won't be able to (UpgradeTag is in Base Application), I will do this. Leaving the comment as a reminder to myself

Reply:
added

Reply:
@JesperSchulz could you please add this comment to internal PR when you create it? You can resolve this comment when you create the PR

Reply:
@onbuyuka, I can see you created the comment yourself, hence resolving this one :-)